tests/cli/pmmAdminSummary_test.js

```diff
@@ -0,0 +1,14 @@
+Feature('pmm-admin summary tests');
+
+Scenario('PMM-T1219 - Verify pmm-admin summary includes targets from vmagent @pmm-admin', async ({ I }) => {
```

Comment:
you have added a new flag, but we don't have a place where we run it right? could you possible use some other tags? we have a tag called as `@cli`

Reply:
I think I forgot to update readme with this and some other tags I added.